.github/workflows/release.yml

```diff
@@ -1,7 +1,6 @@
 name: release
 on:
   push:
-    branches: [main]
```

Comment:
@clairemcginty removing this coz we only need to release on tag push.

Reply:
👍 makes sense!